### ISSUE TITLE
Include a way to view the changelog in Aspine

### ISSUE THREAD

**Issue by psvenk** [Aspine/aspine#174]
It would be nice if users could see (in a non-intrusive way) what new features have been added to Aspine. We could do this by rendering `CHANGELOG.md` to HTML in the backend and serving that via an endpoint which is loaded upon a button click in the info tab (after #157 is merged; that blocker is why this issue is currently on hold). In the lite version build process, we could include a step that runs a script in JS that does this compilation (we might do well to rewrite the build script entirely in JS in that case).

**Comment by tektaxi** [comment]
I think we could use #193 to notify people of changes that are the most relevant to the user. Like "the schedule now works correctly!" or something like that. So along with school things, have Aspine changes listed there.

**Comment by psvenk** [comment]
It may also be good to automate some of the version tagging process. @kdk1616 pointed out that there is duplication between the Git tags, GitHub releases, and the changelog. We could have a script which adds a new entry to the changelog (using user-provided input), increments the version number in `package.json`, runs `npm install`, commits the changes, tags a new version, and pushes the new commit and tag to GitHub. We could then use something like GitHub Actions for handling releases.

**Comment by notrodes** [comment]
We use Github for documenting changes, we link to github from the site already.

**Comment by psvenk** [comment]
@notrodes The point of this feature is that most users have an easy way to see what is new in Aspine, in the same vein as the info tab. Most users will probably not want to take the extra step to come to GitHub, especially because the site is billed as a place for software development (not that this is wrong).

**Comment by notrodes** [comment]
most people know how to click a link and arn't scared to do it
